test/unit/message_test.rb

```diff
@@ -167,6 +174,25 @@ class MessageTest < ActiveSupport::TestCase
     buyer_sent_message.restore_for!(provider)
     assert_not provider_received_message.reload.hidden?
   end
+
+  test "will create messages with only subject and not body" do
```

Comment:
We moved these two tests from the controller tests, but above there are two other new tests that already test the same, so these two are redundant.